.vscode/settings.json

```diff
@@ -1,5 +1,5 @@
 {
-    "python.pythonPath": ".env/bin/python",
+    "python.pythonPath": "/home/tritin.truong/.virtualenvs/eat-2.0/bin/python",
```

Comment:
Shouldn't IDE settings files like `.vscode` be removed entirely from the repo and excluded in the future via `.gitignore`? I believe that's the usual way of handling things like this since it's gonna depend on a each developers local config.

Reply:
@jriddy

No, these files shouldn't be removed because [this manual](https://github.com/opencv/cvat/blob/develop/CONTRIBUTING.md) about how to setup dev environment is based on it. 
So, if we change/delete this settings, the steps from the manual won't work as expected.

Reply:
I also found it useful to recreate the launch scripts on PyCharm.